src/fakeTDLWithMetadata.dfdl.xsd

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xs:schema
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/"
+  xmlns:meta="urn:com.owlcyberdefense.dfdl-test-harness"
```

Comment:
This is problematic. This schema belongs in the test rig or something. Not in fakeTDL.